parsers/ttcn.c

```diff
@@ -933,7 +936,9 @@ static void parseTTCN (void)
 			case T_TEMPLATE:
 				/* A.1.6.1.3 TemplateDef ::= "template" (Type | Signature) ID ... */
 				if (parseType() || parseSignature())
+                {
```

Comment:
Do we need this change for supporting advanced parameterization?
At least this chnage should go to a separated commit.